README.md

```diff
@@ -8,7 +8,18 @@ We all have different backgrounds, beliefs, and points of view. We respect all o
 Treat others as you wish to be treated.
 
 ## Trust and Cooperation
-We are a team, we don't judge rather support each other.
+We are on the same team, we do not judge, rather support each other.
+Everyone can trust that it is safe to raise concerns and speak up if there is a problem. Everyone can trust that it will be taken seriously and it will not backfire.
+Assume the best intent of your colleagues: social distances due to working remotely can give you the opposite impression.
+Everyone can express his opinion without expecting judgment.
+
+We do not judge our teammates' opinions, and we assume their best intentions.
```

Comment:
Isn't this line repeated while merging the first two paragraphs?

Reply:
you're right, it can be removed